conf/fred_hutch.config

```diff
@@ -0,0 +1,83 @@
+params {
+    config_profile_name        = 'Fred Hutch'
+    config_profile_description = 'Fred Hutch Cancer Center HPC profile'
+    config_profile_contact     = 'dgratz@fredhutch.org'
+    config_profile_url         = 'https://sciwiki.fredhutch.org/'
+    // Same as resourceLimits, but for older pipelines
+    max_memory = 700.GB
+    max_cpus = 36
+    max_time = 720.h
+    // Assumes user will provide a scratch dir at runtime
+    scratch = null
```

Comment:
Naming, this might easily be confused with the nextflow `scratch` functionality which is another thing.